dashboard/components/reports.py

```
@@ -0,0 +1,23 @@
+import json
+from typing import Dict, List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)